ee/tabby-ui/app/globals.css

```diff
@@ -54,7 +54,7 @@
     --card: 39 15% 27%;
     --card-foreground: 39 3.2% 99.35%;
 
-    --border: 39 13% 21%;
+    --border: 39 9% 17%;
```

Comment:
Adjust the border color, otherwise the border here is not visible.
<img width="1130" alt="image" src="https://github.com/TabbyML/tabby/assets/17516233/2b7f2d12-366d-4d02-ae1a-4db7dfb15763">